src/Repository/DatabaseConnection.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+namespace BookReviews\Repository;
+
+use \PDO;
+
+/**
+ * Class DatabaseConnection
+ * @package BookReviews\Repository
+ */
+
```

Comment:
You should never have a blank line between the class comment and the class definition